src/frontend/src/flows/addDevice/manage/addDevicePickAlias.ts

```diff
@@ -1,5 +1,9 @@
 import { html, render } from "lit-html";
 import { initLogout, logoutSection } from "../../../components/logout";
+// Regex Pattern for input: All characters, including starting(^) single character or more(+),
```

Comment:
nit: `^` is not the starting character, it's the anchor at the start of the string, but I'm really being picky here...